doc/man7/provider-keymgmt.pod

```diff
@@ -43,6 +43,13 @@ provider-keymgmt - The KEYMGMT library E<lt>-E<gt> provider functions
  /* Discovery of supported operations */
  const char *OP_keymgmt_query_operation_name(int operation_id);
 
+/* Key validation */
+
+int OP_keymgmt_validate_domparams(void *key);
```

Comment:
Nit: "key" should be "domparams", at least if you want to be consistent with the rest

Reply:
shifted into its own section